pgml-cms/docs/introduction/getting-started/connect-your-app.md

```diff
@@ -4,13 +4,13 @@ description: PostgresML is compatible with all standard PostgreSQL clients
 
 # Connect your app
 
-You can connect to your database from any Postgres compatible client. PostgresML is intended to serve in the traditional role of an application database, along with it's extended role as an MLOps platform to make it easy to build and maintain AI applications.
+You can connect to your database from any PostgresSQL-compatible client. PostgresML is intended to serve in the traditional role of an application database, along with it's extended role as an MLOps platform to make it easy to build and maintain AI applications together with your application data.
 
-## Application SDKs
+## SDK
```

Comment:
```suggestion
## Client SDK
```

We've waffled between singular and plural. Changing this also means changing urls for consistency.

Reply:
Not going to write Client SDKs ever again, that wording is just wrong. SDK are only usable on the client by definition.

Reply:
It's going to be singular going forward.